packages/react/src/components/form/Textfield/Textfield.stories.tsx

```diff
@@ -32,30 +32,22 @@ export const WithCharacterCounter: Story = {
   },
 };
 
-export const FullWidth: Story = {
+export const HtmlSize: Story = {
   args: {
     label: 'Label',
-  },
-  parameters: {
-    layout: 'padded',
+    htmlSize: 10,
   },
 };
 
-export const HtmlSize: Story = {
+export const Adornments: Story = {
   args: {
-    label: 'Label',
-    htmlSize: 10,
+    prefix: 'NOK',
+    suffix: 'per måned',
```

Comment:
This is not read out by screen-readers, so its only visual. Maybe shorten it? 
```suggestion
    suffix: 'pr. mnd',
```